README.md

```diff
@@ -254,44 +260,44 @@ public interface FileProcessingWorkflow {
 
     @QueryMethod(name="status")
     String getStatus();
-        
+
     @SignalMethod
-    void retryNow();    
+    void retryNow();
 }
 ```
 ## Starting workflow executions
 
-Given a workflow interface executing a workflow requires initializing a `WorkflowClient` instance, creating 
-a client side stub to the workflow and then calling a method annotated with @WorkflowMethod.
+A workflow interface executing a workflow requires initializing a `WorkflowClient` instance, creating
```

Comment:
I think it changed the meaning. The idea is that to start workflow execution a workflow interface class should be passed to the WorkflowClient which returns a stub that implements passed interface type. Then to start workflow a method on the stub is called.